doc/crud.md

```diff
@@ -268,6 +362,11 @@ This is the simplest case as we don't need to worry about previous revisions.
     CDTDocumentRevision *saved = [datastore createDocumentFromRevision:rev
                                                                  error:&error];
     ```
+    ```swift
+    let rev = CDTDocumentRevision("doc1")
+    rev.body = [....]
```

Comment:
We can't use 3 dots in swift, as its closed range operator, see https://developer.apple.com/library/ios/documentation/Swift/Conceptual/Swift_Programming_Language/BasicOperators.html